symbols/overlay29.yml

```diff
@@ -384,6 +395,22 @@ overlay29:
         Has a special check to return false for Labyrinth Cave B10F (the Gabite boss fight).
         
         return: bool
+    - name: CheckSlowStart
+      address:
+        NA: 0x22F923C
+        EU: 0x22F9C48
+      description: |-
+        Runs a check over all monsters on the field for the ability Slow Start, and lowers the speed of those who have it.
+        
+        No params
+    - name: CheckArtificialWeatherAbilities
```

Comment:
Similar to the Slow Start function, let's call this `TryActivateArtificialWeatherAbilities`